src/visual/aria.ts

```diff
@@ -218,11 +218,13 @@ export default function ariaVisual(ecModel: GlobalModel, api: ExtensionAPI) {
 
                     const middleSeparator = labelModel.get(['data', 'separator', 'middle']);
                     const endSeparator = labelModel.get(['data', 'separator', 'end']);
+                    const columnsToExclude = labelModel.get(['data', 'columnsToExclude']);
                     const dataLabels = [];
                     for (let i = 0; i < data.count(); i++) {
                         if (i < maxDataCnt) {
                             const name = data.getName(i);
-                            const value = data.getValues(i);
+                            const value = !columnsToExclude ? data.getValues(i)
+                                : data.getValues(i).filter((value, j) => !columnsToExclude?.includes(j));
```

Comment:
Use `zrUtil.filter` and `zrUtil.indexOf` instead.

Reply:
[50339df](https://github.com/apache/echarts/pull/20038/commits/50339df94a089d3c055aae8fdc11e2ed9c495fec)